docs/pages/build-reference/variants.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Installing app variants on the same device
+---
+
+When creating [development, preview, and production builds](../eas-json.md#common-use-cases), it's common to want to install one of each build on your device at the same time. This allows you to do development work, preview the next version of your app, and run the production version all on the same device, without needing to uninstall and reinstall the app.
+
+In order to be able to have multiple instances of an app installed on your device, each instance must have a unique Application ID (Android) or Bundle Identifier (iOS).
+
+**If you have a bare project**, you can accomplish this using flavors (Android) and targets (iOS). You can then configure which flavor is used with the `gradleCommand` field on your build profile, and you can configure the target that is used with the `scheme` field for iOS. 
```

Comment:
> To configure which flavor is used, use the `gradleCommand` field on your build profile; to configure which target is used, use the `scheme` field for iOS.

This expands prior sentence in detail, vs following as a next step, so "you can then" is a little confusing.